src/main/java/org/primeframework/mvc/scope/annotation/ManagedCookie.java

```diff
@@ -49,6 +49,14 @@
    */
   boolean encrypt() default true;
 
+  /**
+   * By default, if `encrypt` is true, but an unencrypted cookie is presented by the browser,
```

Comment:
Code formatting in Javadoc comments requires `{@code encrypt}`. Without that, it renders the literal backtick.
![image](https://github.com/user-attachments/assets/ef493751-e6ca-4374-ae06-9653dec39b47)

Another option is `{@link ManagedCookie#encrypt()}`, which will create a link.
![image](https://github.com/user-attachments/assets/4f0b3874-6ee8-404f-91c2-75c592f73352)

Reply:
Good call. Was in back of my mind when doing this but didn't see it through. Fixed.